src/beliefs/k_previous_observations.jl

```diff
@@ -0,0 +1,28 @@
+"""
+Updater that stores the k most recent observations as the belief.
+"""
+struct KMarkovUpdater <: Updater
+    k::Int
+end
+
+function initialize_belief{O}(bu::KMarkovUpdater, obs::O)
+    obs_stacked = Vector{O}(bu.k)
+    for i=1:bu.k
+        obs_stacked[i] = obs
+    end
+    return obs_stacked
+end
+
+function initialize_belief{O}(bu::KMarkovUpdater, obs_vec::Vector{O})

```

Comment:
I don't think the `O` parameter is needed here, right?

Reply:
Should it just be `::Vector{Any}`?
Regardless the parameterization, there might be an ambiguity if O is a vector between:
```julia
initialize_belief{O}(bu::KMarkovUpdater, obs_vec::Vector{O})
initialize_belief{O}(bu::KMarkovUpdater, obs_vec::O)
```
I am not sure how to deal with that